openc3/lib/openc3/microservices/interface_microservice.rb

```diff
@@ -202,6 +202,8 @@ def run
               else
                 next "Interface not connected: #{@interface.name}"
               end
+            rescue WriteRejectError => e
+              next e.message
```

Comment:
No @logger.error because we already printed it interface.rb? If so aren't we double booking other errors by printing in both interface and interface_microservice?

Reply:
I don't want to print a stack trace for this.